src/components/Table/Table.tsx

```diff
@@ -8,10 +8,16 @@ import type {TableSize} from './types';
 
 import './Table.scss';
 
+type VerticalAlignment = 'top' | 'center' | 'bottom';
```

Comment:
It might be better to rename `center` to `middle` (and related CSS modifier as well) because that's how it's called originally `vertical-align: middle`. Also the Table component in uikit uses the same naming.